src/main/java/com/shanebeestudios/skbee/elements/recipe/sections/SecRecipeCooking.java

```diff
@@ -0,0 +1,181 @@
+package com.shanebeestudios.skbee.elements.recipe.sections;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.Section;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import com.shanebeestudios.skbee.SkBee;
+import com.shanebeestudios.skbee.api.recipe.CookingRecipeType;
+import com.shanebeestudios.skbee.api.recipe.RecipeUtil;
+import com.shanebeestudios.skbee.api.util.Util;
+import org.bukkit.Bukkit;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.BlastingRecipe;
+import org.bukkit.inventory.CampfireRecipe;
+import org.bukkit.inventory.CookingRecipe;
+import org.bukkit.inventory.FurnaceRecipe;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.RecipeChoice;
+import org.bukkit.inventory.SmokingRecipe;
+import org.bukkit.inventory.recipe.CookingBookCategory;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.entry.EntryValidator;
+import org.skriptlang.skript.lang.entry.EntryValidator.EntryValidatorBuilder;
+import org.skriptlang.skript.lang.entry.util.ExpressionEntryData;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+@Name("Recipe - Register Cooking Recipe")
+@Description({
+        // Reviewer note: damn there are periods, any suggestions to make this better?
+        "This section allows you to register any cooking recipe add define special properties.",
+        "",
+        "\n'input' = The item the recipe requires as an input to output the result (Required).",
+        "\n'cooktime' = How long the recipe will take to finish cooking before result is given (Optional).",
+        "\n'experience' = The amount of experience gained when the recipe is finished cooking (Optional).",
```

Comment:
I use backticks for these, as it shows up as like a code block on SkriptHub
```suggestion
        "\n`input` = The item the recipe requires as an input to output the result (Required).",
        "\n`cooktime` = How long the recipe will take to finish cooking before result is given (Optional).",
        "\n`experience` = The amount of experience gained when the recipe is finished cooking (Optional).",
```